webrtc.html

```diff
@@ -2875,6 +2891,30 @@ <h2 id="sec.cert-mgmt">Certificate Management</h2>
             </ol>
           </dd>
 
+          <dt>RTCRtpTransceiver addTransceiver((MediaStreamTrack or
+          DOMString) trackOrKind, RTCRtpTransceiverInit init)</dt>
+
+          <dd>
+            <p>Create a new <code><a>RTCRtpTransceiver</a></code> and add it to the
+            collection of transceivers that will be returned
+            by <code><a href="#dom-peerconnection-getreceivers">getReceivers()</a></code>.</p>
+
+	    <p>Adding a transceiver will cause future calls to <code>createOffer</code> to
+	    add an m-line for the corresponding transceiver, as defined in
+	    [[!RTCWEB-JSEP]].
+
+            <p>If a track is passed in, the value of the <code>.sender.track</code> will
+            be set to that track and the MSID and media type generated
+            by <code>createOffer</code> will be that of the track.</p>
+
+            <p>If a kind is passed in, the value of the <code>.sender.track</code> will be
+            null and and media type generated by <code>createOffer</code> will be that of
+            the kind.  The MSID generated by <code>createOffer</code> will be selected by
```

Comment:
According to the MSID spec, this will cause a track to be created at the remote side after an offer/answer exchange. I assume this is required as part of the "warmup" usage.
Don't we want to prevent this by passing "send:false"?